superset/assets/javascripts/explorev2/stores/fields.js

```diff
@@ -55,15 +45,19 @@ export const fields = {
   },
 
   metrics: {
-    type: 'SelectMultipleSortableField',
+    type: 'SelectField',
+    multi: true,
+    freeForm: false,
```

Comment:
i like how you've simplified the select field type to `SelectField` and added `multi` and `freeform` keys here, but i don't see where the changes are in the `SelectField` component to accommodate these new keys. are they in a different PR?